understanding/21/reflow.html

```diff
@@ -12,68 +12,245 @@ <h1>Understanding Reflow</h1>
     <section id="brief">
 			<h2>In brief</h2>
 			<dl>
-				<dt>Goal</dt><dd>Content can be enlarged without increasing line length.</dd>
-				<dt>What to do</dt><dd>Make lines of text reflow within the viewport.</dd>
-        <dt>Why it's important</dt><dd>People who need bigger text find it difficult if they must scroll to read long lines.</dd>
+				<dt>Goal</dt><dd>Content can be enlarged without needing to scroll in two dimensions.</dd>
+				<dt>What to do</dt><dd>Make long lines of text and other applicable content reflow within resized or zoomed in interfaces.</dd>
+        <dt>Why it's important</dt><dd>People who need larger text find it difficult if they need to scroll in two dimensions to read long lines that do not reflow.</dd>
```

Comment:
Again, making something wordier by reusing words that are already in the SC, which arguably does little to  _improve_ understanding while definitely making it less brief.

Reply:
i see your point, i'll look at trying to trim these back down so long as they don't over simplify - which is why i was trying to address these, per the feedback i had received from both people in our group, as well as non-wcag participants

Reply:
My suggestion is to focus on the Understanding document and leave the In Brief alone for now. Remember that the In Brief had full scrutiny by the WG in the past year. The Understanding document itself has not been scrutinized as a whole since publication of 2.1.

Reply:
i'm fine with so long as others who requested these changes agree.

Reply:
I'm just going to ignore the proposed In Brief text revisions for now. I still have concerns with the modified text, but as per my prior statement, it makes sense to leave this new content alone and focus our cycles on material that has not been changed in years.